src/managers/TimestampManager.cpp

```diff
@@ -444,7 +470,6 @@ std::vector<uint8_t> TimestampManager::binaryCompressLocOffsets(
     return finalCompression.bytes;
 }
 
-
 std::vector<int>
 TimestampManager::decompressLocalOffsetList(std::vector<uint8_t> values) {

```

Comment:
En const reference i stedet for at kopiere containeren når denne bliver brugt?

Reply:
Yes, fixed